core/vm/evm.go

```diff
@@ -222,7 +222,7 @@ func (evm *EVM) Call(caller ContractRef, addr common.Address, input []byte, gas
 	}
 
 	if isPrecompile {
```

Comment:
Can we add `isPrecompile` as part of the `CaptureStart/Enter` trace?

Reply:
What's the advantage if added?

Reply:
You know it's a pre-compiled code that was executed, it's for observability of what has executed on the chain. Make it easier than tracking for a given chain what are the pre-compiled addresses.

Reply:
Good idea, I did not think about. Right now `env.precompile` is private, I don't feel that the tracer should re-implement the logic, so I will add `evm.IsPrecompileAddr` to overcome that.

Reply:
This can be used:

https://github.com/ethereum/go-ethereum/blob/5b159498bb73eca1c2864c97811ce0375532762c/core/vm/contracts.go#L151

example:

https://github.com/ethereum/go-ethereum/blob/5b159498bb73eca1c2864c97811ce0375532762c/eth/tracers/native/4byte.go#L85